lib/middleware/karma.js

```diff
@@ -179,6 +179,10 @@ var createKarmaMiddleware = function (
             var filePath = file.path
             var fileExt = path.extname(filePath)
 
+            if (!files.included.hasOwnProperty(i)) {
+              continue
```

Comment:
Perhaps. But let's say they added a new method called randomize to the Array prototype. I don't see any further processing you would want to do to that method. I'd think it should just be skipped.